dev/archery/archery/integration/runner.py

```diff
@@ -126,6 +126,8 @@ def _gold_tests(self, gold_dir):
                             if f.name == name).skip
             except StopIteration:
                 skip = set()
+            if name == 'union' and prefix == '0.17.1':
```

Comment:
This is a minor hack to remove Java from 0.17.1 union gold tests. Not the most elegant way to do it but doing it correctly would be a non-trivial change to archery imho. Thoughts?

Reply:
IMO I think this is OK for now.